velox/exec/HashTable.h

```diff
@@ -864,8 +896,8 @@ class HashTable : public BaseHashTable {
   // many threads can set this.
   std::atomic<bool> hasDuplicates_{false};
 
-  // Offset of next row link for join build side, 0 if none. Copied
-  // from 'rows_'.
+  // Offset of next row link for join build side, 0 if none. Copied from
```

Comment:
What does "Copied from 'rows_'" means here? Does this need to be int64_t?

Reply:
I change to "set from 'rows_'". This is (delta) offset from a pointer so int32_t should be sufficient.